src/cirrus/build.py

```diff
@@ -11,21 +11,43 @@
 """
 import os
 import sys
+from argparse import ArgumentParser
 
 from cirrus.environment import cirrus_home
 from cirrus.configuration import load_configuration, get_pypi_auth
 from fabric.operations import local
 
 
-def main():
+def build_parser(argslist):
     """
-    _main_
+    _build_parser_
+
+    Set up command line parser for the build command
+
+    """
+    parser = ArgumentParser(
+        description = 'git cirrus build'
+    )
+    parser.add_argument('command',nargs='?')
```

Comment:
Make sure you are either running pep8 or have a pep8 plugin running in your editor (space after comma)